### DISCUSSION TITLE
Missing VPNs on "Best Privacy VPN" page

### DISCUSSION THREAD

**Discussion by RePacked** [privacytoolsIO/privacy-tools#7] [General Privacy Forum]
I just noticed that the page only has MullvadVPN listed.

https://www.privacytools.io/privacy-vpn

**Comment by sophiakaile** [comment]
Also try to list these 
https://www.privacyguides.org/en/vpn/
https://wethegeek.com/best-vpn-with-free-trials/
https://www.privateinternetaccess.com/
